Parse/Parse/Internal/PFCoreManager.m

```diff
@@ -82,7 +82,7 @@ - (instancetype)initWithDataSource:(id<PFCoreManagerDataSource>)dataSource {
     _locationManagerAccessQueue = dispatch_queue_create("com.parse.core.locationManager", DISPATCH_QUEUE_SERIAL);
     _controllerAccessQueue = dispatch_queue_create("com.parse.core.controller.accessQueue", DISPATCH_QUEUE_SERIAL);
     _objectLocalIdStoreAccessQueue = dispatch_queue_create("com.parse.core.object.localIdStore", DISPATCH_QUEUE_SERIAL);
-
+    [self objectSubclassingController];
```

Comment:
@ShawnBaek can you help me understand this line? was there a problem that instantiating the subclassing controller here resolves?

Reply:
Sorry  I didn't review details for `Flovilmart PR`

I guess It related that problem you mentioned